src/app/core/components/account-settings-dialog/account-settings-dialog.component.html

```diff
@@ -10,10 +10,10 @@
 		<div class="settings-tabs">
 			<div
 				class="settings-tab"
-				(click)="setTab('account')"
-				[class.active]="activeTab === 'account'"
+				(click)="setTab('email-phone')"
+				[class.active]="activeTab === 'email-phone'"
 			>
-				Account Info
+				Email and PHone
```

Comment:
Oh -- typo here `PHone` vs `Phone`

Still approving the code, but let's fix this before merge